packages/components/src/styles/components/form/layout.scss

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// FORM LAYOUT COMPONENTS
+//
+
+@use "../../mixins/breakpoints" as *;
+
+// FORM
+
+.hds-form {
+  --hds-form-section-max-width: 768px; // default max-width used by FormSection, etc.
+
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+}
+
+// FORM HEADER & SECTION HEADER
+
+.hds-form__header,
+.hds-form__section-header {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+}
+
+// FORM SECTION
+
+.hds-form__section {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+// FIELD GROUP
+// Note: Uses a Layout::Flex component under the hood which includes the main layout styles
+// If responsive behavior is later added to the Layout::Flex component, this behavior should be replaced and
+// the styles should be removed
+
+.hds-form__field-group {
+  // When there are columns, stack them vertically on small screens
+  &.hds-layout-flex--direction-row {
+    // applies to screen widths of 479px & below
```

Comment:
@LilithJames-HDS @didoo 

We may want to consider applying the stacking layout at a larger breakpoint. 

Please test the responsive layout examples to see what you think: https://hds-showcase-git-kristin-hds-4885-form-layout-d00037-hashicorp.vercel.app/components/form#responsiveness

Reply:
Mmm, in code the **media query** is triggered by the viewport being `480px`, if instead you want the stacking to happen when the width **of the form** is `480px` then it's a completely different discussion: it's not anymore a standard viewport breakpoint (viewport/page-level) but a custom **container query** breakpoint (component-level) but the latter comes with a lot of possible downsides. We can discuss about this in the meeting later, though.

Reply:
Update: We discussed and @LilithJames-HDS is going to test a way to replicate the breakpoint behavior in Figma (using a calculation to account for the screen width of an entire page vs. just the Form itself) and then we may revise the breakpoint in code depending on her investigations (likely to the next largest one).